src/backend/api/meals.js

```diff
@@ -1,15 +1,81 @@
+const { query, request, response } = require("express");
 const express = require("express");
 const router = express.Router();
 const knex = require("../database");
 
 router.get("/", async (request, response) => {
   try {
-    // knex syntax for selecting things. Look up the documentation for knex for further info
-    const titles = await knex("meals").select("title");
-    response.json(titles);
+    if ("maxPrice" in request.query) {
+      const maxPrice = request.query.maxPrice
+      const meals = await knex("meal").where("price", "<=", maxPrice)
```

Comment:
I believe the database is called `meals` and not `meal`.
```suggestion
      const meals = await knex("meals").where("price", "<=", maxPrice)
```
Please have a look in `database_dump_meals.sql`